private-cloud/base/aws-iaas/pre_setup.yml

```diff
@@ -130,6 +130,36 @@
         ipaadmin_principal: "{{ freeipa.ipaadmin_user | default(lookup('ansible.builtin.env', 'IPA_USER', default=omit)) }}"
         enable_dns: yes  
 
+    - name: Update the SSSD for caching of Kerberos ticket as files
+      community.general.ini_file:
+        path: /etc/sssd/sssd.conf
+        section: "domain/{{ domain }}"
+        option: "{{ sssd.key }}"
+        value: "{{ sssd.value | string }}"
+      loop: "{{ entries | dict2items }}"
+      loop_control:
+        loop_var: sssd
+        label: "{{ sssd.key }}"
+      vars:
+        entries:
+          krb5_ccname_template: 'FILE:/tmp/krb5cc_%U_XXXXXX'
+
+    - name: Comment default_ccache_name in krb5.conf.d
+      ansible.builtin.replace:
+        dest: /etc/krb5.conf.d/kcm_default_ccache
+        regexp: '({{ krb_item }})'
+        replace: '# \1'
+      loop_control:
+        loop_var: krb_item
+      loop:
+        - '^\[libdefaults\]'
+        - '^\s+default_ccache_name = KCM'
+
+    - name: Restart the SSSD service
```

Comment:
This will always restart? Should we add a guard conditional?

Reply:
Good spot, yes you are right. I've instead used a `restart sssd` handler on these tasks.